ovn-tester/ovn_workload.py

```diff
@@ -423,31 +488,39 @@ def enforce(self):
         self.nbctl.port_group_add_ports(self.pg_def_deny_egr, self.ports)
         self.nbctl.port_group_add_ports(self.pg, self.ports)
 
-    def create_sub_ns(self, ports):
+    def create_sub_ns(self, ports, global_cfg):
         n_sub_pgs = len(self.sub_pg)
         suffix = f'{self.name}_{n_sub_pgs}'
         pg = self.nbctl.port_group_create(f'sub_pg_{suffix}')
         self.nbctl.port_group_add_ports(pg, ports)
         self.sub_pg.append(pg)
-        addr_set = self.nbctl.address_set_create(f'sub_as_{suffix}')
-        self.nbctl.address_set_add_addrs(addr_set,
-                                         [str(p.ip) for p in ports])
-        self.sub_as.append(addr_set)
+        if global_cfg.run_ipv4:
+            addr_set = self.nbctl.address_set_create(f'sub_as_{suffix}')
+            self.nbctl.address_set_add_addrs(addr_set,
+                                             [str(p.ip) for p in ports])
+            self.sub_as.append(addr_set)
+        if global_cfg.run_ipv6:
+            addr_set = self.nbctl.address_set_create(f'sub_as_{suffix}6')
+            self.nbctl.address_set_add_addrs(addr_set,
+                                             [str(p.ip6) for p in ports])
+            self.sub_as.append(addr_set)
         return n_sub_pgs
 
     @ovn_stats.timeit
-    def default_deny(self):
+    def default_deny(self, family=4):
```

Comment:
I think we don't need the default `family=4`.  Let's just pass it explicitly.

Reply:
@dceara ack, I will fix it